package.json

```diff
@@ -35,39 +37,181 @@
+    },
+    "./named-action": {
+      "types": "./build/server/named-action.d.ts",
+      "default": "./build/server/named-action.js"
+    },
+    "./parse-accept-header": {
+      "types": "./build/server/parse-accept-header.d.ts",
+      "default": "./build/server/parse-accept-header.js"
+    },
+    "./preload-route-assets": {
+      "types": "./build/server/preload-route-assets.d.ts",
+      "default": "./build/server/preload-route-assets.js"
+    },
+    "./promise": {
+      "types": "./build/common/promise.d.ts",
+      "default": "./build/common/promise.js"
+    },
+    "./redirect-back": {
+      "types": "./build/server/redirect-back.d.ts",
+      "default": "./build/server/redirect-back.js"
+    },
+    "./respond-to": {
+      "types": "./build/server/respond-to.d.ts",
+      "default": "./build/server/respond-to.js"
+    },
+    "./responses": {
+      "types": "./build/server/responses.d.ts",
+      "default": "./build/server/responses.js"
+    },
+    "./rolling-cookie": {
+      "types": "./build/server/rolling-cookie.d.ts",
+      "default": "./build/server/rolling-cookie.js"
+    },
+    "./safe-redirect": {
+      "types": "./build/server/safe-redirect.d.ts",
+      "default": "./build/server/safe-redirect.js"
+    },
+    "./server-only": {
+      "types": "./build/react/server-only.d.ts",
+      "default": "./build/react/server-only.js"
+    },
+    "./typed-cookie": {
+      "types": "./build/server/typed-cookie.d.ts",
+      "default": "./build/server/typed-cookie.js"
+    },
+    "./typed-session": {
+      "types": "./build/server/typed-session.d.ts",
+      "default": "./build/server/typed-session.js"
+    },
+    "./use-debounce-fetcher": {
+      "types": "./build/react/use-debounce-fetcher.d.ts",
+      "default": "./build/react/use-debounce-fetcher.js"
+    },
+    "./use-delegated-anchors": {
+      "types": "./build/react/use-delegated-anchors.d.ts",
+      "default": "./build/react/use-delegated-anchors.js"
+    },
+    "./use-event-source": {
+      "types": "./build/react/use-event-source.d.ts",
+      "default": "./build/react/use-event-source.js"
+    },
+    "./use-global-navigation-state": {
+      "types": "./build/react/use-global-navigation-state.d.ts",
+      "default": "./build/react/use-global-navigation-state.js"
+    },
+    "./use-hydrated": {
+      "types": "./build/react/use-hydrated.d.ts",
+      "default": "./build/react/use-hydrated.js"
+    },
+    "./use-locales": {
+      "types": "./build/react/use-locales.d.ts",
+      "default": "./build/react/use-locales.js"
+    },
+    "./use-should-hydrate": {
+      "types": "./build/react/use-should-hydrate.d.ts",
+      "default": "./build/react/use-should-hydrate.js"
+    }
+  },
   "peerDependencies": {
-    "@remix-run/react": "^1.10.0",
-    "@remix-run/server-runtime": "^1.10.0",
-    "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
+    "@remix-run/react": "^1.19.1",
+    "@remix-run/router": "^1.7.2",
+    "@remix-run/server-runtime": "^1.19.1",
```

Comment:
What about explicitly setting `@remix-run/cloudflare`, `@remix-run/deno` & `@remix-run/node` here? 
In combination with `peerDependenciesMeta`, this gives you the flexibility to switch your internal code to use the runtimes themselves (if ever needed) instead of purely relying on `@remix-run/server-runtime` without a breaking change